### PR TITLE
v8: update to 10.3.1

### PR DESCRIPTION
**Pull request by raedrizqie**

### CODE REVIEW

File: mingw-w64-v8/PKGBUILD [hunk 1]
```diff
@@ -1,214 +1,198 @@
...
-pkgver=9.1.269.39
-pkgrel=5
+pkgver=10.3.1
+pkgrel=1
 pkgdesc="Fast and modern Javascript engine (mingw-w64)"
 arch=('any')
 mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64' 'clang32')
 url="https://v8.dev"
-license=("BSD")
+license=('spdx:BSD')
```
Comment (lazka) [inline mingw-w64-v8/PKGBUILD:13]: there is no spdx identifier called "BSD"

Comment (ZachBacon) [reply mingw-w64-v8/PKGBUILD:13]: Sometimes you got to read the license block and go to the spdx website to find the proper matching license identifier. We tend to do this for solus too at times.

Comment (raedrizqie) [reply mingw-w64-v8/PKGBUILD]: ah.. sorry

**Comment by raedrizqie** [comment]
@lazka can we merge this? i dont want to rebuild v8 everytime when trying to compile chromium..

**Comment by lazka** [comment]
thanks!

**Comment by jeroen** [comment]
Very nice, thanks!!

**Pull request merged**